webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -38,14 +40,15 @@ export const WorkspaceDetailsTab = {
   Members: 'members',
   ModelRegistry: 'models',
   Projects: 'projects',
+  ResourcePools: 'pools',
```

Comment:
I'm not sure.. These links do look like they should point to workspace projects tab, especially since we are showing the number of projects in this table.